packages/react-centra-checkout/src/ApiClient/index.ts

```diff
@@ -26,10 +26,8 @@ class ApiClient {
       body: ['POST', 'PUT'].includes(method) ? JSON.stringify(data) : undefined,
     })
 
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- TODO: fix this
-    const json = await response.json()
+    const json = (await response.json()) as unknown
```

Comment:
This will result in that callees of `ApiClient.request` will have to parse / validate / cast the returned value.
Doing this will result in more explicitness and enforcing stronger type safety.

Reply:
suggestion: I wonder if we should not just include Matt's `ts-reset` to get some fixes like this.

Reply:
True, will do it in a separate PR, @maeertin!